src/pages/layer-2.js

```diff
@@ -836,7 +775,9 @@ const Layer2Page = ({ data }) => {
       </PaddedContent>
 
       <PaddedContent>
-        <h2>Tools to be effective on layer 2</h2>
+        <h2>
+          <Translation id="layer-2-tools-title" />
+        </h2>
         <TwoColumnContent>
           <Flex50>
             <ProductList
```

Comment:
We should queue up this JSON file for translation too.

<img width="1334" alt="Screenshot 2022-05-30 at 15 39 35" src="https://user-images.githubusercontent.com/62268199/171015500-1f75f5c6-3eb7-4f48-b4fd-6ef887211517.png">